packages/server/src/base-classes/controller.ts

```diff
@@ -0,0 +1,17 @@
+import { Router } from 'express';
+
+import { serverApi } from 'core/api';
+
+import BaseControllerInterface from '../interfaces/base-controller.interface';
+
+export default abstract class BaseController
+  implements BaseControllerInterface {
```

Comment:
Do we really need this new line?

Reply:
Well we do have a basic interface for a BaseController, don't we? If you think an abstract class would be enough I will remove the interface at all

Reply:
UPD: This article http://dotnetpattern.com/typescript-abstract-class suggests that an abstract class can implement an interface. So I will probably leave it, but remove it from the classes that extend `BaseController` what do you think?

Reply:
I meant let's leave it one line:
export default abstract class BaseController implements BaseControllerInterface {

Reply:
Well, that is autocorrected